### ISSUE TITLE
investigate failing Storage2UriMapperJdtImplTest on latest

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-eclipse#1822]
Storage2UriMapperJdtImplTest fails on latest
- what has changed in platform or jdt ?

@szarnekow any idea?
@iloveeclipse are you aware of any changes on eventing for deleted jar files in platform or jdt? e.g. jarpackagefragmentroot etc

**Comment by iloveeclipse** [comment]
No. If test creates new projects, I assume it doesn't wait for all events after that or not properly reacts on them (the utf-8 change that creates .settings in new project probably).

**Comment by cdietrich** [comment]
cannot reproduce locally yet.
will check if test is flaky on server

**Comment by cdietrich** [comment]
Failed again today
